kitsune/users/jinja2/users/auth.html

```diff
@@ -31,41 +31,32 @@
 
     <div class="card feature-padding elevation-03">
       <div class="sumo-auth">
-        <figure class="sumo-auth--logo">
-          <img class="ff-logo" src="{{ webpack_static('sumo/img/FF_Master__icon.svg') }}" alt="{{ _('Firefox logo') }}"/>
-        </figure>
         <div class="sumo-auth--form">
           <div id="login-fxa">
             <article id="login" class="main login-form">
-              <div class="sumo-page-section">
-                <h1 class="sumo-page-heading">{{ _('Log in / Sign up with Firefox Accounts') }}</h1>
-                <p>
-                  {% if aaq_context and not aaq_context.has_subscriptions %}
```

Comment:
I assume by the removal of this block that we don't want to differentiate any more between the login flow for the AAQ and every other case. If this is the case, do we need to do any other cleanup here like the `aaq_context` and the `notification_already_migrated`?

Reply:
Ok, I did some leg work.  There is no need for any alternate text based on AAQ, so that is pulled, along with the already_migrated. We are leaving the AAQ progress bar.  Pushing those up shortly.